dace/transformation/interface/vscode/__main__.py

```diff
@@ -1,6 +1,32 @@
 import json
 from argparse import ArgumentParser
 
+def get_exception_message(exception):
+    if hasattr(exception, 'message'):
+        return exception.message
+    else:
+        return str(exception)
+
+def load_sdfg_from_json(json):
+    # We lazy import SDFGs, not to break cyclic imports, but to avoid any large
+    # delays when booting in daemon mode.
+    from dace.sdfg import SDFG
+    try:
+        sdfg = SDFG.from_json(json)
+        error = None
```

Comment:
what if the SDFG has a validation error internally? Then the SDFG JSON (`json`) has a field called `error`....

Reply:
Also a very good point, I added a feature that will report internal SDFG validation errors back as well.